nemo/collections/nlp/data/datasets/text_classification/text_classification_dataset.py

```diff
@@ -48,14 +48,24 @@ class BertTextClassificationDataset(Dataset):
             If -1, use all dataset. Useful for testing.
     """
 
-    def __init__(self, input_file, max_seq_length, tokenizer, num_samples=-1, shuffle=False, use_cache=False):
+    def __init__(
+        self,
+        input_file,
+        max_seq_length,
+        tokenizer,
+        num_samples=-1,
+        shuffle=False,
+        use_cache=False,
+        do_lower_case=False,
+    ):
         self.input_file = input_file
         self.max_seq_length = max_seq_length
         self.tokenizer = tokenizer
         self.num_samples = num_samples
         self.use_cache = use_cache
         self.shuffle = shuffle
         self.vocab_size = self.tokenizer.tokenizer.vocab_size
+        self.do_lower_case = do_lower_case
```

Comment:
same here, do we need it?

Reply:
Please see my comment for the previous case.